projects/demo/src/modules/components/rating/rating.template.html

```diff
@@ -0,0 +1,139 @@
+<tui-doc-page header="Rating" package="KIT" type="components">
+    <ng-template pageTab>
+        <p i18n>A rating indicates user interest in content</p>
+
+        <tui-doc-example
+            id="base"
+            i18n-heading
+            heading="Basic"
+            [content]="example1"
+        >
+            <tui-rating-example-1></tui-rating-example-1>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="colors"
+            i18n-heading
+            heading="Customization"
+            [content]="example2"
+        >
+            <tui-rating-example-2></tui-rating-example-2>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo [control]="control">
+            <ng-template>
+                <b>Current rating</b>
+                - {{rating.value}}
+                <br />
+
+                <tui-rating
+                    #rating
+                    [min]="min"
+                    [max]="max"
+                    [style.color]="color"
+                    [readOnly]="readOnly"
+                    [iconNormal]="iconNormal"
+                    [iconFilled]="iconFilled"
+                    [formControl]="control"
+                ></tui-rating>
+            </ng-template>
+        </tui-doc-demo>
+        <tui-doc-documentation>
+            <ng-template
+                i18n
+                documentationPropertyMode="input"
+                documentationPropertyType="boolean"
+                documentationPropertyName="readOnly"
+                [(documentationPropertyValue)]="readOnly"
+            ></ng-template>
+            <ng-template
+                i18n
+                documentationPropertyMode="input"
+                documentationPropertyType="boolean"
+                documentationPropertyName="disabled"
+                [documentationPropertyValue]="control.disabled"
+                (documentationPropertyValueChange)="$event ? control.disable() : control.enable()"
+            ></ng-template>
+            <ng-template
+                i18n
+                documentationPropertyType="number"
+                documentationPropertyMode="input"
+                documentationPropertyName="min"
+                [(documentationPropertyValue)]="min"
+            ></ng-template>
+            <ng-template
+                i18n
+                documentationPropertyType="number"
+                documentationPropertyMode="input"
+                documentationPropertyName="max"
+                [(documentationPropertyValue)]="max"
+            ></ng-template>
+            <ng-template
+                i18n
+                documentationPropertyMode="input"
+                documentationPropertyType="string"
+                documentationPropertyName="style.color"
+                [documentationPropertyValues]="colorVariants"
+                (documentationPropertyValueChange)="color = $event"
```

Comment:
Please change this and the rest to a two way binding.

Reply:
fixed